.pre-commit-config.yaml

```diff
@@ -52,3 +52,7 @@ repos:
         files: ^(CHANGES.rst|development.rst|README.rst)$
         language: python
         additional_dependencies: [pygments, restructuredtext_lint]
+-   repo: https://github.com/elidupuis/mirrors-sass-lint
```

Comment:
Can we use [this scss linter instead](https://github.com/pre-commit/mirrors-scss-lint)? It's more actively maintained and from the pre-commit org which makes me feel like it's more likely to stay maintained.

Also, if we move to the above linter, we can specify the revision as a tag instead of a sha. I think it's easier for people to read and understand tags v shas

Reply:
actually i read about https://github.com/pre-commit/mirrors-scss-lint
this uses https://github.com/sds/scss-lint. and this one states "Consider other tools before adopting SCSS-Lint" in its readme. 
so i think this one is worth a discussion, @gnikonorov

Reply:
What do @ssbarnea and @BeyondEvil think? My main concern is the fact that https://github.com/elidupuis/mirrors-sass-lint hasn't been updated since 2017.

Reply:
Some projects reach a high level of maturity and won't see any updates for quite some time, that doesn't mean it's not a valuable project.

How about using [stylint](https://stylelint.io/)?

Reply:
As long as styling is well maintained I have no objections. What do you think @jkowalleck ?

Reply:
not sure.
need to research if [stylint](https://stylelint.io/) works well for the case and how it needs to be configured.

will come back to this topic later.

Reply:
@jkowalleck let's not bloat this PR. Can you create a follow up issue for this so we don't lose track of it? I think the current linter is fine for now

Reply:
created #391 to keep track of it.